resources/original_messages.json

```
@@ -411,6 +411,22 @@
     "message": "Sorry, it looks like Outline is not properly installed. Please try installing it again. If that doesn't work, please submit feedback through the app.",
     "description": "Error message shown when the application cannot connect because the installer could not configure the system or because changes have been made to the system which can only be corrected by running the installer again."
   },
+  "outline_services_installation_confirmation": {
+    "message": "We will set up the Outline VPN, which may require admin permissions. Would you like to proceed?",
+    "desciption": "A prompt message asking for permission whether Outline is allowed to install backend services, because that requires administrative permission."
+  },
+  "outline_services_installation_failed": {
+    "message": "Unfortunately we are not able to set up Outline VPN, please submit feedback through the app.",
+    "desciption": "An error message shown when the user tries to install Outline backend services but failed for some reason."
+  },
+  "outline_services_installed": {
+    "message": "The Outline VPN has been successfully set up, please try again to connect to the server.",
+    "desciption": "An information message shown when the Outline successfully installed all backend services, and we ask the user try to reconnect to the server."
+  },
+  "outline_services_installing": {
+    "message": "Setting up Outline VPN...",
```

Perhaps this for consistency?
```suggestion
    "message": "Setting up the Outline VPN...",
```

Or perhaps
```suggestion
    "message": "Setting up Outline...",
```